docs/docs/usage/scaladoc/settings.md

```diff
@@ -117,3 +117,19 @@ A base URL to use as prefix and add `canonical` URLs to all pages. The canonical
 ##### -siteroot
 
 A directory containing static files from which to generate documentation. Default directory is `./docs`
+
+##### -versions-dictionary-url
```

Comment:
It would be nice to show an example of sbt expression setting the option.

Reply:
I can add this one, but in my opinion it's a basic knowledge how to add setting to sbt.

Reply:
I agree, but I think it’s often good to give copy-pastable examples.